contrib/chart/backstage/values.yaml

```diff
@@ -251,3 +251,5 @@ auth:
   newRelicRestApiKey: r
   travisciAuthToken: fake-travis-ci-auth-token
   pagerdutyToken: h
+  splunkOnCallApiId: h
+  splunkOnCallApiKey: h
```

Comment:
Again, would suggest we may not want to change this globally for the example app, but if needed adding it to the README for the plugin setup instructions?